service/controller/rbac_resource_set.go

```diff
@@ -9,32 +9,41 @@ import (
 	"github.com/giantswarm/operatorkit/resource/wrapper/metricsresource"
 	"github.com/giantswarm/operatorkit/resource/wrapper/retryresource"
 
-	"github.com/giantswarm/rbac-operator/service/controller/resource/test"
+	"github.com/giantswarm/rbac-operator/service/controller/resource/namespaceauth"
 )
 
-type todoResourceSetConfig struct {
+const (
+	nsClusterLabel = "giantswarm.io/cluster"
+	nsOrgLabel     = "giantswarm.io/organization"
```

Comment:
I think that should be put into the label package https://github.com/giantswarm/aws-operator/blob/master/pkg/label/label.go